components/global.css

```diff
@@ -0,0 +1,245 @@
+html, body {
```

Comment:
Does importing css from an external library work well? Or will this need to be duplicated in the app repo?

Reply:
It seems to work well for the imported components as is. I tested it in a template without its own copy. If we want it to be a ui system that standardizes css utilities across other apps we could import global.css from ui-components to replace the local copy.

Reply:
I will add the README what I had to do to get it to work in the app.